cli/command/container/create.go

```diff
@@ -63,6 +65,7 @@ func NewCreateCommand(dockerCli command.Cli) *cobra.Command {
 	flags.StringVar(&opts.name, "name", "", "Assign a name to the container")
 	flags.StringVar(&opts.pull, "pull", PullImageMissing,
 		`Pull image before creating ("`+PullImageAlways+`"|"`+PullImageMissing+`"|"`+PullImageNever+`")`)
+	flags.BoolVarP(&opts.quiet, "quiet", "q", false, "Suppress the pull output")
```

Comment:
I was a bit in doubt on the best description for this (was considering, perhaps we should use the same as `docker swarm ca`, which uses `Suppress progress output`, or `docker push` / `pull` (which use `Suppress verbose output`)

But it looks like we're not consistent anywhere currently, so let's just keep this 👍